test/render-code.js

```diff
@@ -0,0 +1,21 @@
+const test = require('tape');
+const renderCode = require('../lib/render-code');
+const code = getCode();
+
+test('should render code with javascript language', t => {
+    const snippet = renderCode(code, 'javascript');
+
+    t.equal(snippet, '<pre class="language-javascript"><code><span class="token keyword" >const</span> foo <span class="token operator" >=</span> bar<span class="token punctuation" >;</span></code></pre>');
```

Comment:
I'm not sure if the the output needs to be *exactly* that. You could consider also loading `cheerio` in your test so you can check the snippet's `pre` has a class `language-javascript` and that it contains `token`s.